<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrowsingWidget</class>
 <widget class="QWidget" name="BrowsingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="browsingStackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="GamesWidget" name="gamesWidget"/>
     <widget class="StreamsWidget" name="streamsWidget"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GamesWidget</class>
   <extends>QWidget</extends>
   <header>gameswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StreamsWidget</class>
   <extends>QWidget</extends>
   <header>streamswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
